src/verifyRelease.ts

```diff
@@ -95,6 +108,30 @@ async function installNpmPackageVersion(
   shell.exec(`npm install ${packageVersionRef}`, { cwd, fatal: true })
 }
 
+function isNpmPackageAvailable(
+  cwd: string,
+  packageRef: string,
+  packageVersion: string
+): boolean {
+  const checkVersionExistsCmd = shell.exec(
```

Comment:
I'm shocked that `shell.exec` isn't `async`.

Reply:
You can run it async, but it's sync by default.